src/types.ts

```diff
@@ -28,7 +28,7 @@ export interface SchemeData {
 // Every field is optional in this type, to match the reality of starting with
 // a blank form. Mandatory fields are marked in the form UI. Optional string
 // types are encoded as "".
-export interface PipelineScheme {
+export interface PipelineScheme extends PipelineBudget, PipelineTiming {
```

Comment:
This is a difference from #398 that simplifies our lives massively, by keeping the scheme-level data exactly the same. Moving the fields around would be extra backfill complexity, so just keep them at the top level